continuous_delivery_scripts/spdx_report/spdx_helpers.py

```diff
@@ -33,7 +33,7 @@
 logger = logging.getLogger(__name__)
 
 # Copyright similar to the regex defined in flake8-copyright
-COPYRIGHT_PATTERN = r"((?i)Copyright(?i).*$)"
+COPYRIGHT_PATTERN = r"(?i)Copyright.*$"
```

Comment:
I think there was a bug in the regex https://docs.python.org/3.6/library/re.html#re.I but I think the fix is, as follows
```suggestion
COPYRIGHT_PATTERN = r"(?i)Copyright(?-i).*$"
```